web/src/api/schema.d.ts

```diff
@@ -142,8 +142,6 @@ export interface components {
   pathItems: never;
 }
 
-export type $defs = Record<string, never>;
```

Comment:
note: This is rather weird. It seems like people use slightly different versions (even tho it is specified in `package.json` and `yarn.lock`) which produces a different output each time.

Reply:
Turns out I needed to `yarn install`, oops.

Reverted in c8f8aaac7a4c89236e5d5f1198b720be8fcb1b72